dipper/sources/NCBIGene.py

```diff
@@ -258,24 +260,28 @@ def _get_gene_info(self, limit):
                     continue
 
                 if self.class_or_indiv[gene_id] == 'C':
-                    model.addClassToGraph(gene_id, label, gene_type_id, desc)
+                    model.addClassToGraph(gene_id, label, gene_type_id, desc,
+                                          class_category=blv.Gene.value)
                     # NCBI will be the default leader (for non mods),
                     # so we will not add the leader designation here.
                 else:
-                    model.addIndividualToGraph(gene_id, label, gene_type_id, desc)
+                    model.addIndividualToGraph(gene_id, label, gene_type_id, desc,
+                                               ind_category=blv.Gene.value)
```

Comment:
Are there cases we want to use a different cateogry?  See https://github.com/monarch-initiative/dipper/blob/master/translationtable/ncbigene.yaml#L4

NCBI gene also has entites that are closer to phenotypes/diseases, see https://www.ncbi.nlm.nih.gov/gene/261 as an example, curious how these are handled

Reply:
> Are there cases we want to use a different cateogry? See https://github.com/monarch-initiative/dipper/blob/master/translationtable/ncbigene.yaml#L4

@kshefchek good question. I was viewing this categorization (and the biolink model generally) as more of a high-level categorization, so for example a higher-level category like "gene" might be okay here. Saying that, biolink has lower-level categories we can use here, e.g. ["RNA product"](https://biolink.github.io/biolink-model/docs/RnaProduct), where appropriate. 

> NCBI gene also has entites that are closer to phenotypes/diseases, see https://www.ncbi.nlm.nih.gov/gene/261 as an example, curious how these are handled

Right now this code categorizes them as genes, which is slightly wrong I guess, but on the other hand this entry does seem to refer to a gene on 5q35 as well as a phenotype. 

Could add some code to provide finer grain categorization if you think it's advisable

Reply:
If theres something above gene (sequence feature?) I think we could use that for IDs with this type

Reply:
thanks, changed this to GenomicEntity